java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -672,4 +672,124 @@ CompletableFuture<Object[][]> xpending(
+     * @see <a href="https://valkey.io/commands/xinfo-stream/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @return Detailed stream information.
+     * @example <pre>{@code
+     * // example of using the API:
+     * Map<String, Object> response = client.xinfoStreamFull("myStream").get();
+     * // the response contains data in the following format:
+     * Map<String, Object> data = Map.of(
+     *     "length", 4L,
+     *     "radix-tree-keys", 1L,
+     *     "radix-tree-nodes", 2L,
+     *     "last-generated-id", "1719877599564-0",
+     *     "max-deleted-entry-id", "0-0",
+     *     "entries-added", 4L,
+     *     "recorded-first-entry-id", "1719710679916-0",
+     *     "entries", Map.of(
+     *         "1719710679916-0",
+     *         new String[][] {
+     *             { "foo", "bar" },
+     *             { "foo", "bar2" },
+     *             { "some_field", "some_value" }
+     *         },
+     *         "1719710688676-0",
+     *         new String[][] {
+     *             { "foo", "bar2" },
+     *         // Up to 10 entries here
+     *     }),
+     *     "groups", new Map[] {
+     *         Map.of(
+     *             "name", "mygroup",
+     *             "last-delivered-id", "1719710688676-0",
+     *             "entries-read", 2L,
+     *             "lag", 0L,
+     *             "pel-count", 2L,
+     *             "pending", new Object[][] { {
+     *                     "1719710679916-0",
+     *                     "Alice",
+     *                     1719710707260L,
+     *                     1L,
+     *                 }, {
+     *                     "1719710688676-0",
+     *                     "Alice",
+     *                     1719710718373L,
+     *                     1L
+     *                 } },
+     *             "consumers", new Map[] {
+     *                "name", "Alice",
+     *                "seen-time", 1719710718373L,
+     *                "active-time", 1719710718373L,
+     *                "pel-count", 2L,
+     *                "pending", new Object[][] { {
+     *                        "1719710679916-0",
+     *                        1719710707260L,
+     *                        1L,
+     *                    }, {
+     *                        "1719710688676-0",
+     *                        1719710718373L,
+     *                        1L
+     *                    } }
+     *             })
+     * });
+     * }</pre>
+     */
+    CompletableFuture<Map<String, Object>> xinfoStreamFull(String key);
+
+    /**
+     * Returns verbose information about the stream stored at key <code>key</code>.
+     *
+     * @since Redis 6.0 and above.
+     * @see <a href="https://valkey.io/commands/xinfo-stream/">valkey.io</a> for details.
+     * @param key The key of the stream.
+     * @param count The number of stream and PEL entries that are returned. Value of <code>0</code> means that all entries will be returned.
```

Comment:
Should we include the warning that the Redis doc mentions about potentially long execution time if 0 is provided or not necessary?